packages/components/src/components/domainproperties/dataset/DatasetDesignerPanels.tsx

```diff
@@ -232,44 +289,94 @@ export class DatasetDesignerPanelImpl extends React.PureComponent<Props & Inject
         const { setSubmitting } = this.props;
         const { model, fileImportData } = this.state;
 
-        saveDomain(model.domain, model.getDomainKind(), model.getOptions(), model.name)
-            .then(response => {
-                this.setState(
-                    produce((draft: Draft<State>) => {
-                        const updatedModel = draft.model;
-                        updatedModel.exception = undefined;
-                        updatedModel.domain = response;
-                    }),
-                    () => {
-                        // If we're importing Dataset file data, import file contents
-                        if (fileImportData) {
-                            this.handleFileImport();
-                        } else {
-                            const { model } = this.state;
+        let keyPropIndex;
+        let visitPropIndex;
+
+        if (this._participantId && this._sequenceNum) {
```

Comment:
I don't think all cases of dataset creation will need to have both of these properties in order to save. I assume this is what is blocking the case where you want to create a dataset manually instead of from a file creation.

Reply:
agree, this is changed in the next commit